boogiestats/boogie_ui/views.py

```diff
@@ -240,6 +240,10 @@ def get_context_data(self, **kwargs):
         song_hash = self.kwargs["song_hash"]
         context["song"] = Song.get_or_404(hash=song_hash)
         context["num_highscores"] = Score.objects.filter(song_id=song_hash, is_top=True).count()
+        if hasattr(self.request.user, "player"):
+            context["my_scores"] = Score.objects.filter(
+                song_id=song_hash, player_id=self.request.user.player.id
```

Comment:
This will let django take care of figuring out what are the models' pk fields and make the whole more concise:

```
        song_hash = self.kwargs["song_hash"]
        song = Song.get_or_404(hash=song_hash)
        context["song"] = song
        context["num_highscores"] = song.scores.filter(is_top=True).count()
        if hasattr(self.request.user, "player"):
            context["my_scores"] = song.scores.filter(player=self.request.user.player).count()
```

Reply:
fixed